pkg/build/controller/strategy/sti_test.go

```diff
@@ -77,13 +77,17 @@ func testSTICreateBuildPod(t *testing.T, rootAllowed bool) {
 		t.Errorf("Expected never, got %#v", actual.Spec.RestartPolicy)
 	}
 	// strategy ENV is whitelisted into the container environment, and not all
-	// the values are allowed, so only expect 9 not 10 values.
-	expectedEnvCount := 9
+	// the values are allowed, so only expect 10 not 11 values.
+	expectedEnvCount := 10
```

Comment:
the set of whitelisted env variables (which is just BUILD_LOGLEVEL) plus the set of env variables we explicilty add (which has gone up by one since we now consistently add SOURCE_URI whereas before it only got added to some build strategies)